### PR TITLE
[chore](thirdparty)We need to issue an error when starting FE without setting the Java home environment

### PR DESCRIPTION
**Pull request by xyfsjq**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: bin/start_fe.sh [hunk 1]
```diff
@@ -108,7 +108,12 @@ if [[ -e "${DORIS_HOME}/bin/palo_env.sh" ]]; then
 fi
 
 if [[ -z "${JAVA_HOME}" ]]; then
-    JAVA="$(command -v java)"
+    if ! command -v java &>/dev/null; then
```
Comment (yiguolei) [inline bin/start_fe.sh:111]: please add some comment here， why do we need write code like this.
Or the other developer may refactor code and rewrite this code.

Comment (xyfsjq) [reply bin/start_fe.sh:111]: Due to the customer's machine not being configured with Java home, starting FE will result in startup failure, no error prompts, and logs. In this case, when the FE cannot start, it is necessary to prompt an error message indicating that it has not yet configured Java home.

Comment (adonis0147) [reply bin/start_fe.sh:113]: Hi @xyfsjq , would you show the errors here if we don't set `JAVA_HOME` before starting FE?

By the way, if it was caused by the environment variable `JAVA_HOME` which is unset, we should check the variable `JAVA_HOME` straightforwardly.

**Comment by xiaokang** [comment]
run buildall

**Review by adonis0147 — CHANGES_REQUESTED** [review]

**Review by adonis0147 — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**